Berksfile

```diff
@@ -5,8 +5,8 @@ metadata
 
 cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
 cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
-cookbook 'hops', github: "logicalclocks/hops-hadoop-chef", branch: "master"
-cookbook 'ndb', github: "logicalclocks/ndb-chef", branch: "master"
+cookbook 'hops', github: "kouzant/hops-hadoop-chef", branch: "HOPSWORKS-1825"
```

Comment:
Is this intentional.

Reply:
Yes for testing it. I'll change it back ;)